src/angular-modal-service.js

```diff
@@ -103,14 +103,15 @@
             //  Parse the modal HTML into a DOM element (in template form).
             var modalElementTemplate = angular.element(template);
 
+            //  Create the controller, explicitly specifying the scope to use.
+            var modalController = $controller(controllerName, inputs);
+            
             //  Compile then link the template element, building the actual element.
             //  Set the $element on the inputs so that it can be injected if required.
             var linkFn = $compile(modalElementTemplate);
             var modalElement = linkFn(modalScope);
             inputs.$element = modalElement;
 
-            //  Create the controller, explicitly specifying the scope to use.
-            var modalController = $controller(controllerName, inputs);
```

Comment:
Hi! After that, my modal controller died, because $element not found. 
My controller: 
'LightWidgetProductController',
    ['$element', 'close',
    function(el, close
I'm bad or this commit bad?